src/index.js

```diff
@@ -2,66 +2,56 @@ let listenAction
 
 export default {
   bind(el, binding) {
-    const params = binding.value || {}
+    const params = binding.value || {},
+      stickyTop = params.stickyTop || 0,
+      zIndex = params.zIndex || 1000,
+      elStyle = el.style
```

Comment:
I know it is... This whole PR is just a new proposal.

I use this notation because the js compiler will stop and set all those variables once in my code.

When you have multiples const/let/var, the compiler will stop for every const/let/var you have.

So, in this example, you have 1 call vs. 3 calls /stop for the compiler to do